slidetoact/src/main/res/values/attrs.xml

```diff
@@ -17,6 +17,7 @@
         <attr name="slider_locked" format="boolean" />
         <attr name="slider_icon" format="reference" />
         <attr name="rotate_icon" format="boolean" />
+        <attr name="animate_complete" format="boolean" />
```

Comment:
Can you rename it to `animate_completion`?

Reply:
Done.